### PR TITLE
case BUGS:560 . wrapping dependencyManager::get<ddeFacetracker>with ifdef

### PR DESCRIPTION
**Pull request by amerhifi**
https://highfidelity.atlassian.net/browse/BUGZ-560

Adding to this bug. The dependency manager log was going a bit crazy each frame that it was trying to resolve the ddeFacetracker. This pr wraps that call with #ifdef HAVE_DDE which prevents the warning output from the dependency manager, on instance when we don't have an instance of ddefacetracker.

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -7227,7 +7229,7 @@ void Application::nodeKilled(SharedNodePointer node) {
     _octreeProcessor.nodeKilled(node);
 
     _entityEditSender.nodeKilled(node);
-
+
```
Comment (huffman) [inline interface/src/Application.cpp]: Can we get rid of this whitespace change?

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19857/HighFidelity-Beta-PR15712-b57e486.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19857/HighFidelity-Quest-Beta-PR15712-b57e486.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19857/HighFidelity-Beta-PR15712-b57e486.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19857/HighFidelity-Beta-Interface-PR15712-b57e486.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19857/HighFidelity-Beta-PR15712-b57e486.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19857/HighFidelity-Beta-Interface-PR15712-b57e486.exe

File: interface/src/Application.cpp (hunk 1 above)
Comment (amerhifi) [reply interface/src/Application.cpp]: looking at the raw file I don't see the space. I

**Pull request merged**